### PR TITLE
[UM-730] Update repository setup

### PR DESCRIPTION
**Pull request by kirill-panoply**
# Description

- Add CI/CD workflow
- Update project structure (move code into `src` and tests into `test`)
- Misc updates (`PULL_REQUEST_TEMPLATE.md`, `README.md`, `.gitignore`, `package.json`)

### CODE REVIEW

**Comment by kirill-panoply** [comment]
@ilia-panoply I don't have any permissions in this repository

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+on:
+  push:
+    branches: [master]
+
+  pull_request:
+    branches: [master]
+
+env:
+  node_version: 10.14
```
Comment (ilia-panoply) [inline .github/workflows/main.yml:11]: @kirill-panoply please change it to node 14

**Review by ilia-panoply — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (kirill-panoply) [reply .github/workflows/main.yml:11]: I don't think we should do it since we're using this package in applications which are running on node 10.14

Comment (ilia-panoply) [reply .github/workflows/main.yml:11]: @OmriGLevy is working on node 14 migration those days

Comment (kirill-panoply) [reply .github/workflows/main.yml:11]: Yeah so let's update node version here after it will be done.

**Pull request merged**